storage/pruning/fullHistoryTriePruningStorer.go

```diff
@@ -20,38 +19,40 @@ func NewFullHistoryTriePruningStorer(args *FullHistoryStorerArgs) (*fullHistoryT
 }
 
 func initFullHistoryTriePruningStorer(args *FullHistoryStorerArgs, shardId string) (*fullHistoryTriePruningStorer, error) {
-	err := checkArgs(args.StorerArgs)
-	if err != nil {
-		return nil, err
-	}
-
-	activePersisters, persistersMapByEpoch, err := initPersistersInEpoch(args.StorerArgs, shardId)
+	fhps, err := initFullHistoryPruningStorer(args, shardId)
 	if err != nil {
 		return nil, err
 	}
 
-	ps, err := initPruningStorer(args.StorerArgs, shardId, activePersisters, persistersMapByEpoch)
-	if err != nil {
-		return nil, err
+	tps := &triePruningStorer{
+		PruningStorer: fhps.PruningStorer,
 	}
+	fhps.PruningStorer.extendPersisterLifeHandler = tps.extendPersisterLife
+
+	return &fullHistoryTriePruningStorer{
+		triePruningStorer:         tps,
+		storerWithEpochOperations: fhps,
+		args:                      args.StorerArgs,
+		shardId:                   shardId,
+	}, nil
+}
 
-	tps := &triePruningStorer{ps}
-	ps.extendPersisterLifeHandler = tps.extendPersisterLife
-	tps.registerHandler(args.Notifier)
+// GetFromEpoch will call GetFromEpoch from the underlying FullHistoryPruningStorer
+func (fhtps *fullHistoryTriePruningStorer) GetFromEpoch(key []byte, epoch uint32) ([]byte, error) {
+	return fhtps.storerWithEpochOperations.GetFromEpoch(key, epoch)
+}
 
-	if args.NumOfOldActivePersisters < 1 || args.NumOfOldActivePersisters > math.MaxInt32 {
-		return nil, storage.ErrInvalidNumberOfOldPersisters
-	}
+// GetBulkFromEpoch will call GetBulkFromEpoch from the underlying FullHistoryPruningStorer
+func (fhtps *fullHistoryTriePruningStorer) GetBulkFromEpoch(keys [][]byte, epoch uint32) ([]storageCore.KeyValuePair, error) {
+	return fhtps.storerWithEpochOperations.GetBulkFromEpoch(keys, epoch)
+}
 
-	fhps := &fullHistoryTriePruningStorer{
-		triePruningStorer: tps,
-		args:              args.StorerArgs,
-		shardId:           shardId,
-	}
-	fhps.oldEpochsActivePersistersCache, err = lrucache.NewCacheWithEviction(int(args.NumOfOldActivePersisters), fhps.onEvicted)
-	if err != nil {
-		return nil, err
-	}
+// PutInEpoch will call PutInEpoch from the underlying FullHistoryPruningStorer
```

Comment:
could rename all these comments by following this pattern:
`// <Func> will call the same function from the underlying FullHistoryPruningStorer`

Reply:
Done.